examples/gcoap/Makefile

```diff
@@ -27,6 +27,22 @@ USEMODULE += shell
 USEMODULE += shell_commands
 USEMODULE += ps
 
+# DTLS configuration
+GCOAP_ENABLE_DTLS ?= 0
+ifeq (1,$(GCOAP_ENABLE_DTLS))
+	CFLAGS += -DCONFIG_GCOAP_ENABLE_DTLS=1
+	CFLAGS += -DCONFIG_GCOAP_DTLS_SESSIONS_MAX=1
+	# UDP Port to use (20220 is default for DTLS).
+	DTLS_PORT ?= 20220
+	CFLAGS += -DDTLS_DEFAULT_PORT=$(DTLS_PORT)
```

Comment:
I think changing the `DTLS_DEFAULT_PORT` does not make sense here. One might also want to use other applications using DTLS. Rather, a `GCOAPS_DEFAULT_PORT` should be introduced.